addons/test_mail/tests/test_performance.py

```diff
@@ -1606,11 +1607,13 @@ def get_bus_params():
                                     ),
                                 },
                                 {
+                                    "display_name": "Ignasse Inbox",
```

Comment:
So globally this new entry in the dict is useless... Would say to revert and fix only the failing use case in JS. Not sure how you ended up with this, but it is globally over engineered: it adds noise in all sent data for a niche use case. Fixes should be simple and as local as possible.

Reply:
Makes no sense, display_name was not sent prior to this PR. What you call display_name is just name or email. Suggested recipients do not need the display name, would make nos ense to display "Acme Inc. Raoulette <raoulette@acme.inc>": you are on a document, you just want to have a list of people that are going to receive your mail.